### PR TITLE
EdgeCraftRAG: Add E2E test cases for EdgeCraftRAG - local LLM and vllm

### PR DESCRIPTION
**Pull request by rui2zhang**
## Description

Add E2E test cases for EdgeCraftRAG - local LLM and vllm

## Issues

N/A

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

langchain-core, llama-index

## Tests

`./test_e2e_local_llm_on_intel_cpu.sh`
image

./test_e2e_vllm_on_intel_arc.sh
image

### CODE REVIEW

File: EdgeCraftRAG/tests/test_e2e_local_llm_on_intel_cpu.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```
Comment (chensuyue) [inline EdgeCraftRAG/tests/test_e2e_local_llm_on_intel_cpu.sh:1]: `EdgeCraftRAG/tests/test_e2e_local_llm_on_intel_cpu.sh` --> `EdgeCraftRAG/tests/test_compose_on_xeon.sh`

File: EdgeCraftRAG/tests/test_e2e_vllm_on_intel_arc.sh [hunk 2]
```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
```
Comment (chensuyue) [inline EdgeCraftRAG/tests/test_e2e_vllm_on_intel_arc.sh:1]: `EdgeCraftRAG/tests/test_e2e_vllm_on_intel_arc.sh` --> `EdgeCraftRAG/tests/test_compose_vllm_on_arc.sh`

File: EdgeCraftRAG/tests/test_e2e_local_llm_on_intel_cpu.sh (hunk 1 above)
Comment (rui2zhang) [reply EdgeCraftRAG/tests/test_e2e_local_llm_on_intel_cpu.sh]: Ranamed

File: EdgeCraftRAG/tests/test_e2e_vllm_on_intel_arc.sh (hunk 2 above)
Comment (rui2zhang) [reply EdgeCraftRAG/tests/test_e2e_vllm_on_intel_arc.sh]: Ranamed

File: EdgeCraftRAG/tests/test_compose_on_arc.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ip_address=$(hostname -I | awk '{print $1}')
+HOST_IP=$ip_address
+
+COMPOSE_FILE="compose.yaml"
+EC_RAG_SERVICE_PORT=16010
+#MODEL_PATH="$WORKPATH/models"
+MODEL_PATH="/home/media/models"
+HF_ENDPOINT=https://hf-mirror.com
+
+HUGGINGFACEHUB_API_TOKEN="your token"
```
Comment (chensuyue) [inline EdgeCraftRAG/tests/test_compose_on_arc.sh:27]: Please remove this line.

Comment (rui2zhang) [reply EdgeCraftRAG/tests/test_compose_on_arc.sh]: modified

**Review by chensuyue — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**